lib/ai_bot/personas/persona.rb

```diff
@@ -223,6 +232,20 @@ def tool_instance(parsed_function, bot_user:, llm:, context:)
           )
         end
 
+        def strip_quotes(value)
+          if value.is_a?(String)
+            if value.start_with?('"') && value.end_with?('"')
+              value = value[1..-2]
+            elsif value.start_with?("'") && value.end_with?("'")
+              value = value[1..-2]
+            else
```

Comment:
Not sure about your use case here, but should we attempt to target more quotes?

```ruby
if value.match?(/\A[‘’“”‹›«»"'].*[‘’“”‹›«»"']\z/)
    value = value[1..-2]
  end
```

Reply:
chat gpt suggest this regex:

```ruby
def strip_opposite_quotes(value)
  # Use regex to strip matching opposite quotes at the start and end of a string
  value.gsub(/\A(?:["'](.*?)["']|“(.*?)”|‘(.*?)’|«(.*?)»|「(.*?)」)\z/, '\1\2\3\4\5')
end
```

Might be too convulted for our case, but I have been bitten multiple times by these non "classic" quotes, and prefer to mention them.

Reply:
I hear you, but this is pretty safe as is cause this is all llm generated function calls so the odds are pretty low it will mix in something incorrect here. possible but unlikely.